DQMOffline/L1Trigger/src/L1TStage2CaloLayer2Offline.cc

```diff
@@ -386,17 +380,24 @@ void L1TStage2CaloLayer2Offline::fillJets(edm::Event const& e, const unsigned in
   double recoEta = leadingRecoJet.eta();
   double recoPhi = leadingRecoJet.phi();
 
-  double l1Et = closestL1Jet.et();
-  double l1Eta = closestL1Jet.eta();
-  double l1Phi = closestL1Jet.phi();
+  double l1Et = foundMatch ? closestL1Jet.et() : 0;
+  double l1Eta = foundMatch ? closestL1Jet.eta() : 9999;
+  double l1Phi = foundMatch ? closestL1Jet.phi() : 9999;
 
   // if no reco value, relative resolution does not make sense -> sort to overflow
   double outOfBounds = 9999;
   double resolutionEt = recoEt > 0 ? (l1Et - recoEt) / recoEt : outOfBounds;
-  double resolutionEta = std::abs(recoEta) > 0 ? (l1Eta - recoEta) / recoEta : outOfBounds;
-  double resolutionPhi = std::abs(recoPhi) > 0 ? (l1Phi - recoPhi) / recoPhi : outOfBounds;
+  double resolutionEta = l1Eta - recoEta;
+  double resolutionPhi = l1Phi - recoPhi;
```

Comment:
This delta needs to wrap around at pi. The delta cannot be larger than pi since then they are back to back.

Reply:
Will change to `reco::deltaPhi(l1Phi, recoPhi) `

Reply:
BTW: This will give you sensible values if no matched jet was found, shall I do
`l1Phi < 9999 ? reco::deltaPhi(l1Phi, recoPhi) : 9999` instead?

Reply:
But then you would not fill the histogram if no match is found. So I guess it does not matter in the end. It would be cleaner though.

Reply:
Yes, you are right. I will add `outOfBounds` just in case someone in the future decides to still fill the plots ;). Otherwise it is an easy to overlook bug with `reco::deltaPhi`.